### DISCUSSION TITLE
Evaluation results

### DISCUSSION THREAD

**Discussion by SherlockHolmes221** [fredzzhang/upt#52] [Q&A] [answered]
the mAP results are the same as the official hico-det code? Thanks

**Comment by fredzzhang** [comment, marked as answer]
Hi @SherlockHolmes221,

I assume you are referring to the evaluation utilities provided by [Chao et al.](https://github.com/ywchao/ho-rcnn). And yes the results should be identical. In the [launch_template.sh](https://github.com/fredzzhang/upt/blob/main/launch_template.sh), we have provided commands to cache detections so you can evaluate with the Matlab code. Feel free to try both evaluation methods.

Fred.

**Comment by SherlockHolmes221** [comment]
Thx
